ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -359,7 +366,14 @@ - (void)requestTokenImpl:(ADAuthenticationCallback)completionBlock
 #if !AD_BROKER
              [ADAuthenticationRequest releaseExclusionLock];
 #endif
-
+             
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS1131)